app/webpack/stylesheets/gto/components/_confirm_modal.scss

```diff
@@ -0,0 +1,19 @@
+.confirm-modal__content--main {
+  max-width: 250px;
+  > p:first-child {
+    margin-top: 0;
+  }
+  margin-bottom: 10px;
+}
+.confirm-modal__actions {
+  display: flex;
+  width: 100%;
+  justify-content: space-between;
+  align-items: center;
+}
+.confirm-modal__action-item {
```

Comment:
note to reader - i have no problems leaving empty styles in css.  this is an indication to me (and others) that there is a hook to apply some styles.  

thoughts?